package.json

```diff
@@ -4,31 +4,35 @@
   "private": true,
   "dependencies": {
     "@adobe/aio-lib-core-logging": "^2.0.0",
+    "@adobe/aio-lib-files": "^3.0.1",
+    "@adobe/aio-lib-state": "^2.1.2",
     "@adobe/aio-sdk": "^3.0.0",
-    "openwhisk": "^3.21.7",
+    "@adobe/exc-app": "^0.2.17",
     "@azure/msal-node": "^1.17.2",
-    "node-fetch": "^2.6.0"
+    "cloudevents": "^4.0.2",
+    "node-fetch": "^2.6.0",
+    "openwhisk": "^3.21.7",
+    "regenerator-runtime": "^0.13.5",
+    "uuid": "^8.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.21.3",
     "@babel/eslint-parser": "^7.21.3",
     "@openwhisk/wskdebug": "^1.3.0",
-    "eslint": "^8.57.0",
+    "eslint": "^8.36.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jest": "^27.2.3",
+    "eslint-plugin-jest": "^26.5.3",
     "eslint-plugin-jsdoc": "^39.3.3",
     "eslint-plugin-n": "^15.2.3",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
-    "jest": "^29"
+    "jest": "^27.2.4"
```

Comment:
Why downgrade? Here and few other places..

Reply:
will update this.. thanks